src/app/app-routing.module.ts

```diff
@@ -4,7 +4,7 @@ import { Routes, RouterModule } from '@angular/router';
 
 
 const routes: Routes = [
-  {path:'', component:LoginComponent}
+  { path: 'login', loadChildren: () => import('./authentication/authentication.module').then(m => m.AuthenticationModule) }
```

Comment:
path should be '"" so that if you logon to home http://localhost:4200/ you'll get directed to the auth module